ui/src/components/AccordionSection/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import * as OutlineIcons from '@heroicons/react/24/outline';
+import * as Components from '@/components';
+import * as Framer from 'framer-motion';
+
+type Props = {
+    children: React.ReactNode;
+    id: string;
+    title: string;
+};
+
+const AccordionSection: React.FC<Props> = (props) => {
+    const [expanded, setExpanded] = useState(true);
+
+    const toggleId = `${props.id}-toggle`;
+    const childId = `${props.id}-items`;
+
+    return (
+        <Framer.AnimatePresence>
+            <Framer.motion.section
```

Comment:
Ah Framer Motion. This is used in one very specific place on Hedd and I couldn't for the life of me figure out the magic behind it. Tried to reproduce it several times myself using html/js/css but couldn't get anywhere near!

Reply:
It's cool but I'm finding it a pain with jest at the moment - the animations throw it off sometimes as it thinks there was an unintended update to the component. But it is possible to get around